ts/defs.bzl

```diff
@@ -331,60 +331,42 @@ def ts_project(
     typings_out_dir = declaration_dir if declaration_dir else out_dir
     tsbuildinfo_path = ts_build_info_file if ts_build_info_file else name + ".tsbuildinfo"
 
+    # Flags for what is emitted and which tool is emitting them
+    emit_js = not no_emit and (not emit_declaration_only)
```

Comment:
nit: `not no_emit and not emit_declaration_only` or `(not no_emit) and (not emit_declaration_only)` reads better